client/extensions/woocommerce/components/share-widget/index.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * External dependencies
+ */
+import config from 'config';
+import { localize } from 'i18n-calypso';
+import React, { Component, PropTypes } from 'react';
+import url from 'url';
+
+/**
+ * Internal dependencies
+ */
+import SocialLogo from 'social-logos';
+
+class ShareWidget extends Component {
+	static propTypes = {
+		text: PropTypes.string,
+		title: PropTypes.string,
+		urlToShare: PropTypes.string,
+	}
+
+	renderServiceIcons = () => {
+		const { translate, urlToShare } = this.props;
+		const services = [
+			{
+				icon: 'facebook',
+				urlProperties: {
+					scheme: 'https',
+					hostname: 'www.facebook.com',
+					pathname: '/sharer.php',
+					query: {
+						u: urlToShare,
+						app_id: config( 'facebook_api_key' ),
```

Comment:
I chose facebook_api_key based on this existing implementation - https://github.com/Automattic/wp-calypso/blob/a70587157b33783ba4d5e0e4a1e136680d459d51/client/blocks/reader-share/index.jsx#L56